build.gradle.kts

```diff
@@ -43,6 +42,7 @@ repositories {
     wireDetektRulesRepo()
     google()
     mavenCentral()
+    maven(url = "https://oss.sonatype.org/content/repositories/snapshots")
```

Comment:
afaik this will cause issues when releasing to fdroid, lets be careful and wait for the fix to be merged first